docs/source/docker/advanced/datapool.rst

```diff
@@ -1,101 +1,107 @@
-model, the datapool contains the checkpoint `checkpoint_latest.ckpt` which was 
-used to generate the embeddings.
-
-The datapool is located in the `shared` directory. In general, it is a directory
-with the following structure:
+In this example we will do the following steps:
 
+#. Create a worker run to process a cloud bucket with 3 videos
+#. Add 2 more videos to the same bucket
+#. Run the worker with the same config again to use the datapool feature
 
-.. code-block:: bash
 
-    # example of a datapool
-    datapool/
-    +--- datapool_latest.csv
-    +--- checkpoint_latest.ckpt
-    +--- history/
-  
-The files `datapool_latest.csv` and `checkpoint_latest.csv` are updated after every
-run of the Lightly Docker. The history folder contains the previous versions of 
-the datapool. This feature is meant to prevent accidental overrides and can be 
-deactivated from the command-line (see `Usage`_ for more information).
+Here we show the content of the bucket before running the Lightly Worker for the
+first time.
 
-Usage
----------------
+.. code-block:: console
 
-To **initialize** a datapool, simply pass the name of the datapool as an argument
-to your docker run command and sample from a dataset as always. The Lightly Docker
-will automatically create a datapool directory and populate it with the required
-files.
+    videos/
+    |-- campus4-c0.avi
+    |-- passageway1-c1.avi
+    `-- terrace1-c0.avi
 
-.. note:: To use the datapool feature, the Lightly Docker requires write access
-          to a shared directory. This directory can be passed with the `-v` flag.
+Now we can run the following code to select a subset based on the 
+`min_distance` stopping condition. 
 
-.. code-block:: console
+.. literalinclude:: ./code_examples/python_run_datapool_example.py
+  :linenos:
+  :language: python
 
-   docker run --gpus all --rm -it \
-      -v {INPUT_DIR}:/home/input_dir:ro \
-      -v {SHARED_DIR}:/home/shared_dir \
-      -v {OUTPUT_DIR}:/home/output_dir \
-      lightly/worker:latest \
-      token=MYAWESOMETOKEN \
-      append_weak_labels=False \
-      stopping_condition.min_distance=0.1 \
-      datapool.name=my_datapool
+After running the code we have to make sure we have a running Lightly worker 
+to process the job.
+We can start a Lightly Worker using the following command
 
+.. code-block:: console
 
-To **append** to your datapool, pass the name of an existing datapool as an argument.
-The Lightly Docker will read the embeddings and filenames from the existing pool and
-consider them during selection. Then, it will update the datapool and checkpoint files.
+  docker run --rm --gpus all -it \
+    -v /docker-output:/home/output_dir lightly/worker:latest \
+    token=YOUR_TOKEN  worker.worker_id=YOUR_WORKER_ID
 
-.. note:: You can't change the dimension of the embeddings once the datapool has
-          been initialized so choose carefully!
+Afer processing the dataset once we add more videos. The bucket now looks
```

Comment:
Afer processing the dataset once we add more videos -> After we have processed the initial data and created a dataset, we've collected more data and our bucket now looks like this